dlt/destinations/impl/bigquery/bigquery.py

```diff
@@ -254,30 +247,24 @@ def start_file_load(self, table: TTableSchema, file_path: str, load_id: str) ->
                         )
 
                     job = job_cls(
-                        table,
                         file_path,
                         self.config,  # type: ignore
-                        self.schema,
                         destination_state(),
                         functools.partial(_streaming_load, self.sql_client),
                         [],
                     )
                 else:
                     job = BigQueryLoadJob(
-                        FileStorage.get_file_name_from_file_path(file_path),
-                        self._create_load_job(table, file_path),
+                        file_path,
                         self.config.http_timeout,
                         self.config.retry_deadline,
                     )
+            # TODO: this section may not be needed, BigQueryLoadJob will not through errors here and the streaming insert i don't know
             except api_core_exceptions.GoogleAPICallError as gace:
```

Comment:
this is 100% not needed and no calls to big query should be made from here. AFAIK you create jobs before sending them to threads and without calling `with` client so the connection in `sql_client` is closed

Reply:
yes that is correct, now that you mention it I have removed it. I think the test coverage for the streaming insert is not super good. we can improve that in the future maybe too, but generally speaking it is protected by the custom destination state mechanism, so I think it should be good enough for now.